tests/integration/pipeline/test_bootstrap_command.py

```diff
@@ -0,0 +1,163 @@
+from unittest import skipIf
+
+from parameterized import parameterized
+
+from samcli.lib.pipeline.bootstrap.stage import Stage
+from tests.integration.pipeline.base import BootstrapIntegBase
+from tests.testing_utils import run_command_with_input, RUNNING_ON_CI, RUNNING_TEST_FOR_MASTER_ON_CI, RUN_BY_CANARY
+
+# bootstrap tests require credentials and CI/CD will only add credentials to the env if the PR is from the same repo.
+# This is to restrict tests to run outside of CI/CD, when the branch is not master or tests are not run by Canary
+SKIP_DEPLOY_TESTS = RUNNING_ON_CI and RUNNING_TEST_FOR_MASTER_ON_CI and not RUN_BY_CANARY
+
+
+@skipIf(SKIP_DEPLOY_TESTS, "Skip bootstrap tests in CI/CD only")
+class TestBootstrap(BootstrapIntegBase):
+    @parameterized.expand([("create_ecr_repo",), (False,)])
+    def test_interactive_with_no_resources_provided(self, create_ecr_repo: bool):
+        stage_name = self._method_to_stage_name(self.id())
+        self.stack_names = [Stage._get_stack_name(stage_name)]
+
+        bootstrap_command_list = self.get_bootstrap_command_list(interactive=True)
+
+        inputs = [
+            stage_name,
+            "",  # pipeline user
```

Comment:
just want to make sure about the behavior when there is a default value from pipelineconfig.toml file and we provide an empty string here. Will the final value be empty string or the one provided in pipelineconfig.toml ?

Reply:
good question, I think it should be the one in pipelineconfig.toml.

Reply:
well, I asked this because if there is a value in pipelineconfig.toml then this test case will fail as the pipeline-user is provided not created. However, as you said, it is highly unlikely to run the integration tests from a directory that already has a `pipelineconfig.toml` file. Let's not handle this case but we need to add a comment about it.

Reply:
Yeah it is unlikely. I don't think we need comment about the environment. It is a reasonable assumption that the test environment is clean. Otherwise we need to have comments in a lot of places, like samconfig.toml will affect package or deploy.